playbooks/nfs.yml

```diff
@@ -38,3 +38,4 @@
         clients:  "{{ 'nfs_clients' in groups and inventory_hostname in groups['nfs_clients'] }}" 
       nfs_server: "{{ groups['nfs_servers'] | first }}"
       nfs_export: "{{ hostvars[groups['nfs_servers'] | first]['nfs_export'] }}"
+      nfs_client_mnt_point: "{{ nfs_client_mnt_point }}"
```

Comment:
There's risks about making this variable - lots of things (eg, user home dir paths) depend on it.  Just saying.

Reply:
It now points to "jasmin_homedir" instead